yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -434,6 +446,129 @@ revision 2019-01-02 {
+        had a backwards-compatible change at a particular module or sub-module revision</t>
+        <t hangText="node-editorial-change:">Indicates a specific schema node
+        had an editorial change at a particular module or sub-module revision. 
+        The meaning of an editorial change is as per YANG Semver 
+		<xref target="I-D.ietf-netmod-yang-semver"/></t>
+    </list>
+    </t>
+    <t>When a compatibility scope extension is added to a schema node in a sub-module,
+    the revision indicated for the extension statement is that of the sub-module.
+    </t>
+    <t>Adding, modifying or removing any of the schema node compatibility extension
+      statements is considered to be a BC change.
+    </t>
+    <t>The following example illustrates the schema node compatibility extensions:
+      <figure>
+        <artwork>
+          container some-stuff {
+            leaf used-to-be-a-string {
+              rev:node-non-backwards-compatible-change "3.0.0";
+              type uint32;
+            }
+            leaf fixed-my-description-typo {
+              rev:node-editorial-change "1.4.1";
+              type string;
+              description "This description used to have a typo."
+            }
+            list sir-changed-a-lot {
+              rev:node-editorial-change "3.0.0";
+              rev:node-backwards-compatible-change "2.3.0";
+              rev:node-backwards-compatible-change "1.2.1_non_compatible";
+              description "a list of stuff";
+              ordered-by user;
+              key "foo";
+              leaf foo {
+                type string;
+              }
+              leaf thing {
+                type uint8;
+              }
+            }
+        </artwork>
+      </figure>
+    </t>
+    <t>Note that an individual schema node may have a backwards compatible change
+    in a revision that is non-backwards-compatible (e.g. some other node changed
+    in a non-backwards-compatible fashion in that particular revision).
+    </t>
+    <t>If changes are ported from one branch of YANG model revisions to another
+    branch, care must be taken with any schema node compatibilty extensions. 
+    A simple copy-n-paste should not be used. The extensions 
+    may incorrectly reference a revision that is not in the history of the 
+    new revision. Further, the annotations might not apply depending on what 
+    the history is like in that new branch (e.g., an NBC change
+    that is ported might not be an NBC change in the new branch).
+    Schema node compatiblity extensions should not be copied over to the new 
+    branch. Instead, the changes should be 
+    considered as completely new on the new branch, and any compatibility information
+    should be generated from scratch.
+    </t>
+    <t>TBD: Is the format right in the example above ? quotes and semi colon ?</t>
+    <t>TBD: Are we going to define when to add and not add these tags ? e.g.
```

Comment:
We didn't finish debating this on the call today, but one thing we thought we should add:
Nothing about compatibility of a node/statement can be inferred from teh absence of the granular compatibility statements.

Reply:
updated in the PR